packages/flutter_tools/lib/src/base/logger.dart

```diff
@@ -235,13 +237,13 @@ class VerboseLogger extends Logger {
     final String indentMessage = message.replaceAll('\n', '\n$indent');
 
     if (type == _LogType.error) {
-      stderr.writeln(prefix + terminal.bolden(indentMessage));
+      parent.printError(prefix + terminal.bolden(indentMessage));
       if (stackTrace != null)
-        stderr.writeln(indent + stackTrace.toString().replaceAll('\n', '\n$indent'));
+        parent.printError(indent + stackTrace.toString().replaceAll('\n', '\n$indent'));
     } else if (type == _LogType.status) {
-      print(prefix + terminal.bolden(indentMessage));
+      parent.printStatus(prefix + terminal.bolden(indentMessage));
     } else {
-      print(prefix + indentMessage);
+      parent.printStatus(prefix + indentMessage);
```

Comment:
FYI, https://github.com/flutter/flutter/issues/10021 (I didn't have the heart to send you any further down this hole).